packages/patternlab/styleguide/source/assets/js/modules/skipLinkTarget.js

```diff
@@ -18,37 +25,46 @@ document.querySelectorAll(".ma__figure__skip-link").forEach(link => {
   });
 });
 
-// Hide the skipTargetAnchor/e.target as users move to next element.
+// Hide the anchor/e.target as users move to next element.
 // Check a keypress action with tab or arrow keys,
 // which indicates that users attempts to leave the active anchor to another element.
-// No need to display the active anchor at this point until its corresponding link gets clicked again.
-document.querySelectorAll(".ma__figure__skip-link_target a").forEach(anchor => {
-
+// No need to display the anchor at this point until its corresponding link gets clicked again.
+document.querySelectorAll(".js-skiplink-target > a").forEach((anchor) => {
+  // When using arrow keys to move to another element, focus stays on the anchor and it remains visible.
+  // Hide the anchor as they move to next element and remove focus from it.
+  // Note: In JAWS and NVDA, the anchor remains visible until another element gets focus.
+  //       Unable to capture e.key with screen readers running.  Leave this for future enhancement.
   anchor.addEventListener("keydown", (e) => {
-
     if (e.target.getAttribute("tabindex") === "0") {
-      if (e.key === "Tab" || e.key === "ArrowUp" || e.key === "ArrowRight" || e.key === "ArrowDown" || e.key === "ArrowLeft" ) {
-        if (e.target === document.activeElement) {
-          e.target.classList.remove("visible");
-          e.target.setAttribute("tabindex", -1);
-        }
+      switch (e.key) {
+        case "Tab":
+        case "ArrowUp":
+        case "ArrowDown":
+        case "ArrowLeft":
+        case "ArrowRight":
+          skipLinkAnchor(e);
+          e.target.blur();
+          break;
       }
     }
   });
 
-  // For screen readers:
-  // When using arrow keys to move to another element, focus stays on the anchor.
-  // This results the anchor remains visible. Hide the anchor when another element gets focus to make it less distracting
-  // to screen reader users.
+  // For keyboard users, the anchor goes invisible as it loses focus.
   anchor.addEventListener("blur", (e) => {
-
-    if ((e.target !== document.activeElement) && (e.target.getAttribute("tabindex") === "0")) {
-      e.target.classList.remove("visible");
-      e.target.setAttribute("tabindex", -1);
+    if (
+      e.target !== document.activeElement &&
+      e.target.getAttribute("tabindex") === "0"
+    ) {
+      skipLinkAnchor(e);
     }
   });
 });
 
+function skipLinkAnchor (e) {
+  e.target.classList.toggle("visible");
```

Comment:
```suggestion
  e.target.classList.remove("visible");
```
Instead of toggle, can we just specify remove here?